products/shopware.md

```diff
@@ -0,0 +1,184 @@
+---
+title: Shopware
+category: server-app
+tags: php-runtime
+iconSlug: shopware
+permalink: /shopware
+# alternate_urls: /
+versionCommand: php bin/console --version
+# releasePolicyLink: https://shopware.com/
+changelogTemplate: "https://github.com/shopware/shopware/blob/{{'__LATEST__'}}/CHANGELOG.md"
+releaseDateColumn: true
+activeSupportColumn: Bug fix maintenance
+eolColumn: Security maintenance
+# extendedSupportColumn: Shopware end of software support
```

Comment:
I think it's better to document it in advance, and add a column for the same.

```suggestion
extendedSupportColumn: Extended Support
```

Not sure about the wording, since safefive seems to use all of LTS/Commercial/Extended. But we need to be consistent and clear that this isn't official support from Shopware itself.